picard/ui/mainwindow.py

```diff
@@ -792,9 +792,12 @@ def play_file(self):
 
     def open_folder(self):
         files = self.tagger.get_files_from_objects(self.selected_objects)
+        opened = []
```

Comment:
You should use a `set()` here, we don't care about order, elements are unique by definition, and search is done in O(1) vs O(n).